packaging/package/package.py

```diff
@@ -16,11 +16,11 @@ def create_arg_parser():
     parser.add_argument('input_dir', help='path to directory of connector files to package and sign')
     parser.add_argument('-v', '--verbose', dest='verbose', action='store_true', help='verbose output', required=False)
     parser.add_argument('-l', '--log', dest='log_path', help='path of logging output', default='packaging_log.txt')
-    parser.add_argument('--validate_only', dest='validate_only', action='store_true', help='runs package validation steps only', required=False)
+    parser.add_argument('--validate-only', dest='validate_only', action='store_true', help='runs package validation steps only', required=False)
```

Comment:
I think this change is already in 625b1bc which was committed directly to the `packaging` branch.

if you do `git pull origin packaging` it should show up in your branch. It's fine that you have it here, won't cause a merge conflict.